### DISCUSSION TITLE
Installation on RPi3 "file not found"

### DISCUSSION THREAD

**Discussion by Higgins91** [flobz/psa_car_controller#383] [General]
Hello,
I can't install the software. I followed the procedure for Linux, I have python 3.7.3 installed, the command "pip3 install psa-car-controller" worked well but when I run the command "psa-car-controller --web- conf" I get an error: "bash: psa-car-controller: command not found"
I searched for "psa-car-controller" but it doesn't exist.
Is it possible to have a detailed step-by-step installation procedure?
Thanks a lot.
Alexandre

Bonjour,
je n'arrive pas à installer le logiciel. J'ai suivi la procédure pour Linux, j'ai python 3.7.3 d'installé, la commande "pip3 install psa-car-controller" a bien fonctionnée mais quand je lance la commande "psa-car-controller --web-conf" j'obtiens une erreur: "bash: psa-car-controller : commande introuvable"
J'ai recherché "psa-car-controller" mais il n'hexiste pas. 
Est-il possible d'avoir une procédure détaillée d'installation pas à pas ?
Merci beaucoup.
Alexandre

**Comment by cchartrain** [comment]
Peut-être ajouter le "$HOME/.local/bin" (ou vérifier où a été placé psa-car-controller) au path. Pendant l'installation, il y a des warning à propos de ça.
Moi, c'est à l'étape d'après que je coince 👍 Toujours à l'exécution de psa-car-controller --web-conf : 
`
2022-09-27 17:55:22,856 :: ERROR :: Bad config
WARNING: dev bundles requested with serve_locally=False.
This is not supported, switching to serve_locally=True
2022-09-27 17:55:28,767 :: INFO :: update_data
`

**Comment by Higgins91** [comment]
merci pour ta réponse, j'ai ajouté le dossier d'install dans le PATH.
pareil que toi, quand je lance la commade "psa-car-controller --web-conf", j'ai une erreur "BAD CONFIG" puis le paramètre serve_locally qui passe à TRUE et enfin le message update_data qui s'affiche et plus rien...
![IMG_20221008_114533](https://user-images.githubusercontent.com/55664294/194701283-ad541671-3ce8-40ea-a784-101d41294b50.jpg)
c'est très frustrant :(

je regarde du coté de la configuration, il lui faut peut-etre des infos mais sans trop savoir quoi chercher c'est pas évident...
